charts/qiskit-serverless/values.yaml

```diff
@@ -59,7 +59,7 @@ gateway:
       type: ClusterIP
       port: 8000
     ray:
-      nodeImage: "icr.io/quantum-public/qiskit-serverless/ray-node:0.14.0-py310"
+      nodeImage: "icr.io/quantum-public/qiskit-serverless/ray-node:0.14.0-py311"
```

Comment:
Should we just start naming it: `ray-node:0.14.0` and drop `py311`?

Reply:
works for me